src/views/TaxFiling/AdjustedGrossIncomeSection.tsx

```diff
@@ -0,0 +1,121 @@
+import { Table } from '@components/Table/Table'
+import { TableHead } from '@components/TableHead/TableHead'
+import { TableBody } from '@components/TableBody/TableBody'
+import { TableRow } from '@components/TableRow/TableRow'
+import { TableCell } from '@components/TableCell/TableCell'
+import { TableCellAlign } from '@internal-types/table'
+import { Heading } from '@ui/Typography/Heading'
+import { Span } from '@ui/Typography/Text'
+import { Tooltip, TooltipTrigger, TooltipContent } from '@ui/Tooltip/Tooltip'
+import { convertNumberToCurrency } from '@utils/format'
+import { TaxEstimateDetailSection } from './TaxEstimateDetailSection'
+import type { AdjustedGrossIncomeItem } from './TaxEstimate'
+
+interface AdjustedGrossIncomeSectionProps {
+  items: AdjustedGrossIncomeItem[]
+}
+
+export const AdjustedGrossIncomeSection = ({
+  items,
+}: AdjustedGrossIncomeSectionProps) => {
+  const totalItem = items.find(item => item.isTotal)
+  const totalAmount = totalItem?.amount
+
+  return (
+    <TaxEstimateDetailSection title='Adjusted Gross Income' totalAmount={totalAmount}>
+      <div className='Layer__tax-estimate__agi-table'>
+        <Table borderCollapse='collapse'>
+          <TableHead>
+            <TableRow isHeadRow rowKey='agi-header'>
+              <TableCell isHeaderCell>Label</TableCell>
+              <TableCell isHeaderCell align={TableCellAlign.RIGHT}>Amount</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {items.map((item) => {
+              if (item.isSeparator) {
+                return (
+                  <TableRow
+                    key={item.id}
+                    rowKey={item.id}
+                  >
+                    <TableCell colSpan={2} className='Layer__tax-estimate__separator-row' />
+                  </TableRow>
+                )
+              }
+              return (
+                <TableRow
+                  key={item.id}
+                  rowKey={item.id}
+                >
+                  <TableCell className={item.isDeduction ? 'Layer__tax-estimate__deduction-item' : ''}>
+                    {item.isTotal
+                      ? (
+                        <Span size='md' className='Layer__tax-estimate__agi-total-label'>
+                          {item.label}
+                        </Span>
+                      )
+                      : (
+                        <Span
+                          size='md'
+                          className={item.isDeduction ? 'Layer__tax-estimate__deduction-item' : ''}
```

Comment:
`Layer__tax-estimate__deduction-item` is just adding padding - we can do that directly on the span without the class name